backend/sql/initial_schema.sql

```diff
@@ -0,0 +1,103 @@
+create table project
+(
+    project_id  bigserial not null primary key,
+    name        text      not null unique,
+    description text      null
+);
+
+create table test_methodology
+(
+    test_methodology_id bigserial     not null primary key,
+    name                text          not null unique,
+    description         text          null,
+    confluence_link     varchar(1000) null
+);
+
+-- Corresponds to an EXCEL file
+create table experiment
+(
+    experiment_id  bigserial                not null primary key,
+    project_id     bigint                   not null,
+    methodology_id bigint                   not null,
+    -- This is input from the user
+    name           text                     null,
+    meta           text                     null,
+    -- This is fallback name to show if name is not specified
+    raw_file_name  text                     not null,
+    upload_date    timestamp with time zone not null,
+
+    constraint belongs_to_project
+        foreign key (project_id) references project (project_id)
+            on update cascade
+            on delete cascade,
+
+    constraint has_methodology
+        foreign key (methodology_id) references test_methodology (test_methodology_id)
+            -- Methodology_id is immutable
+            on update no action
+            -- Prevent removing methodology if there is an experiment having such methodology
+            on delete no action
+);
+
+create table measurement
+(
+    measurement_id bigserial        not null primary key,
+    experiment_id  bigint           not null,
+    -- Either compound id or name
+    compound_id    text             not null,
+    concentration  double precision not null,
+    signal         double precision not null,
+
+    constraint belongs_to_experiment
+        foreign key (experiment_id) references experiment (experiment_id)
+            on delete cascade
+            on update cascade
+);
+
+create table analysis_method
+(
+    analysis_method_id bigserial not null primary key,
+    description        text      null,
+    -- Analysis-specific parameters
+    parameters         json      null
+);
+
+create table analysis
+(
+    analysis_id        bigserial          not null primary key,
```

Comment:
This is the most important table for the first phase of the project, because we want to get the science right quickly and spend majority of time on figuring out good UX for data systematisation and access.

So let's not try to be overly smart and use a very simple, very standard approach to desigining tables like this.

![MEASUREMENT_T_ANALYSIS](https://user-images.githubusercontent.com/45364419/107960208-3f267b80-6f9c-11eb-96cb-08c9d77906e6.png)

Reply:
we have `analysis_removed_measurements` table. do you want us to original measurements not removed one?

Reply:
Sorry, you're right.

Reply:
Just call make sure that you don't have there anything except for `analysis_id` and `measurement_id` and you PRIMARY KEY over it.